robot_smach_states/src/robot_smach_states/util/startup.py

```diff
@@ -25,10 +25,12 @@
 from robot_skills.util.robot_constructor import robot_constructor
 
 
-def startup(statemachine_creator, initial_state=None, robot_name='', challenge_name=None, argv=sys.argv):
+def startup(statemachine_creator, statemachine_args = (), initial_state=None, robot_name='', challenge_name=None, argv=sys.argv):
```

Comment:
The changes made in this file were a fix for challenge cleanup. With the new designator iterator I don't know if they are still required.

Reply:
Maybe not but seems like a reasonable addition which doesn't break anything judging by a grep of the usages. All uses specify keyword args for everything that is a keyword arg here, so cannot break anything.